### ISSUE TITLE
Feature Request: Desktop App

### ISSUE THREAD

**Issue by amirabiri** [joeferner/redis-commander#446]
Running Redis commander inside Docker is limiting. I installed it via npm and run it and solved several issues I had. However this npm approach also has its drawbacks.

I wonder if a move towards packaging as a desktop app (using something like [Electron](https://www.electronjs.org/) for example) is valid. It will make redis-commander a much more powerful Redis client for me.

**Comment by sseide** [comment]
Hi - when you build an electron (or similar) app around it i am interested to see it - when it is running we can discuss on how to integrate it into redis-commander as an additional way of using it (i've you like...). But it will not replace the standalone web-ui and docker version as they are needed to run it servers in a centralized way.

Another question - what kind of problems did you have with the docker setup? Something we can solve?

**Comment by amirabiri** [comment]
> what kind of problems did you have with the docker setup? Something we can solve?

Not really. It is related to the Docker networking and our company VPN. I just couldn't connect to our Redis servers. But even with local development it is annoying to keep having to remember that hostnames are from within the Docker network.

I just view it like any other DB client that I use. The other clients I use (mongo 3T, Datagrip) - are native apps from within which I can manage both localhost DBs and remote DBs - even if I shut down Docker for Mac. I can also recommend them as tools for our QA teams without having to install them on our actual QA envs without them having to run Docker. Anyone can just download the DB GUI, grab the credentials from the KB - and connect. It's simple.

With a Dockerized approach all of this becomes much more complicated.
